src/main/java/in/org/projecteka/hdaf/link/link/model/PatientLinkReferenceRequestHIP.java

```diff
@@ -14,6 +16,7 @@
 @NoArgsConstructor
 @Setter
 public class PatientLinkReferenceRequestHIP {
-    private String consentManagerUserId;
-    private Patient patient;
+    private String transactionId;
+    @JsonProperty("patient")
+    private PatientInHIP patientInHIP;
```

Comment:
```suggestion
    private Patient patientInHIP;
```
Can have different package for it.